docs/event-bus/08-01-configure-default-channelprovisioner.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Subscription
+type: Configure default ClusterChannelProvisioner
+---
+
+Kyma comes with NATSS as its default ClusterChannelProvisioner(See [default-channel-webhook](../../resources/knative-eventing/charts/knative-eventing/templates/eventing.yaml) ConfigMap). Other than NATSS, one can use any default channel provisioner of one's choice. Following are some provisioners which can be used.
```

Comment:
```suggestion
Kyma comes with NATSS as its default ClusterChannelProvisioner.  You can see the configuration details in the [default-channel-webhook](../../resources/knative-eventing/charts/knative-eventing/templates/eventing.yaml) ConfigMap . You can, however, use other default channel provisioner of your choice. An example can be the in-memory-channel provisioner.
```
Maybe we can also add the configuration part (as below) for the user to be sure where it is and what it looks like? Then in the next point we can reference it (instead of linking to the same place). 

```
apiVersion: v1
data:
  default-channel-config: |
    clusterdefault:
      apiversion: eventing.knative.dev/v1alpha1
      kind: ClusterChannelProvisioner
      name: natss
kind: ConfigMap
metadata:
  name: default-channel-webhook
  namespace: knative-eventing
  labels:
    "helm.sh/hook": "pre-install"

```